bundles/org.openhab.binding.pixometer/src/main/java/org/openhab/binding/pixometer/internal/PixometerBindingConstants.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.pixometer.internal;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+import jersey.repackaged.com.google.common.collect.Sets;
+
+/**
+ * The {@link PixometerBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Jerome Luckenbach - Initial contribution
+ */
+@NonNullByDefault
+public class PixometerBindingConstants {
+
+    private static final String BINDING_ID = "pixometer";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID BRIDGE_THING_TYPE = new ThingTypeUID(BINDING_ID, "apiservice");
+    public static final ThingTypeUID THING_TYPE_ENERGYMETER = new ThingTypeUID(BINDING_ID, "energymeter");
+    public static final ThingTypeUID THING_TYPE_GASMETER = new ThingTypeUID(BINDING_ID, "gasmeter");
+    public static final ThingTypeUID THING_TYPE_WATERMETER = new ThingTypeUID(BINDING_ID, "watermeter");
+
+    public static final Set<ThingTypeUID> BRIDGE_THING_TYPES_UIDS = Collections.singleton(BRIDGE_THING_TYPE);
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Sets.newHashSet(THING_TYPE_ENERGYMETER,
```

Comment:
Remove all dependencies on `jersey.repackaged.com.google.common.collect...`. It refers to refer to a repackaged version of Google Guava (Which I assume is available via the additional classpath entries I guess? The .classpath file doesn't look clean, and has hardcoded paths. I would assume entries should be removed). The Guava library itself has been removed with the migration to bnd as it's a big library and most features are nowadays standard available since Java 8. And can explain why you might have gotten errors when migrating this binding. In htis case replace this code with:  
```suggestion
    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS =
    Stream.of(THING_TYPE_ENERGYMETER, THING_TYPE_GASMETER, THING_TYPE_WATERMETER).collect(Collectors.toSet());
```